app/views/user_mailer/welcome.html.haml

```diff
@@ -70,7 +70,7 @@
 %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
   %tr
     %td.email-extra-td
-      = render 'application/mailer/feature', key: 'feature_control', feature_title: t('user_mailer.welcome.feature_control_title'), feature_text: t('user_mailer.welcome.feature_control'), text_first_on_desktop: true
-      = render 'application/mailer/feature', key: 'feature_audience', feature_title: t('user_mailer.welcome.feature_audience_title'), feature_text: t('user_mailer.welcome.feature_audience'), text_first_on_desktop: false
-      = render 'application/mailer/feature', key: 'feature_moderation', feature_title: t('user_mailer.welcome.feature_moderation_title'), feature_text: t('user_mailer.welcome.feature_moderation'), text_first_on_desktop: true
-      = render 'application/mailer/feature', key: 'feature_creativity', feature_title: t('user_mailer.welcome.feature_creativity_title'), feature_text: t('user_mailer.welcome.feature_creativity'), text_first_on_desktop: false
+      = render 'application/mailer/feature', feature: 'control', text_first_on_desktop: true
+      = render 'application/mailer/feature', feature: 'audience', text_first_on_desktop: false
+      = render 'application/mailer/feature', feature: 'moderation', text_first_on_desktop: true
+      = render 'application/mailer/feature', feature: 'creativity', text_first_on_desktop: false
```

Comment:
This makes it so that `i18n-tasks` cannot know which translation keys are in use… but those are still in use in the text version.